### ISSUE TITLE
Blender crashes on scene bake: Assertion failed: _phi.isIndexInRange(i, j, k)

### ISSUE THREAD

**Issue by vmxeo** [rlguy/Blender-FLIP-Fluids-Beta#232]
Blender is consistently crashing when baking a sim:
```
Step time: 0.0416667 (100% of frame)

02-Apr-2018 18h19m59s BEGIN       Update Obstacle Objects
02-Apr-2018 18h20m02s COMPLETE    Update Obstacle Objects
02-Apr-2018 18h20m02s BEGIN       Update Liquid Level Set
02-Apr-2018 18h20m02s COMPLETE    Update Liquid Level Set
02-Apr-2018 18h20m02s BEGIN       Advect Velocity Field
02-Apr-2018 18h20m02s COMPLETE    Advect Velocity Field
02-Apr-2018 18h20m02s BEGIN       Save Velocity Field
02-Apr-2018 18h20m02s COMPLETE    Save Velocity Field
02-Apr-2018 18h20m02s BEGIN       Apply Body Forces
02-Apr-2018 18h20m02s COMPLETE    Apply Body Forces
02-Apr-2018 18h20m02s BEGIN       Solve Pressure System
02-Apr-2018 18h20m02s COMPLETE    Solve Pressure System
02-Apr-2018 18h20m02s BEGIN       Constrain Velocity Field
02-Apr-2018 18h20m02s COMPLETE    Constrain Velocity Field
02-Apr-2018 18h20m02s BEGIN       Update Marker Particle Velocities
02-Apr-2018 18h20m02s COMPLETE    Update Marker Particle Velocities
02-Apr-2018 18h20m02s BEGIN       Delete Saved Velocity Field
02-Apr-2018 18h20m02s COMPLETE    Delete Saved Velocity Field
02-Apr-2018 18h20m02s BEGIN       Advect Marker Particles
02-Apr-2018 18h20m02s COMPLETE    Advect Marker Particles
02-Apr-2018 18h20m02s BEGIN       Update Fluid Objects
02-Apr-2018 18h20m02s COMPLETE    Update Fluid Objects
02-Apr-2018 18h20m02s BEGIN       Generate Output Data
Assertion failed: _phi.isIndexInRange(i, j, k), file C:\Users\ryanl\Documents\development\bl_flip_fluid\src\engine\meshlevelset.cpp, function get, line 92

This application has requested the Runtime to terminate it in an unusual way.
Please contact the application's support team for more information.
```

Interestingly, this does not happen if _Mesh Around Obstacles_ is enabled.

**Comment by vmxeo** [comment]
Just to clarify, this is happening consistently in this one file. I haven't had this problem with other scenes.

**Comment by rlguy** [comment]
Thanks for letting me know! This error means that the simulator is accessing memory that is out of bounds, which is a bug. Would it be possible to attach the .blend file for testing?

**Comment by vmxeo** [comment]
File is attached....

[Fluid Test 07.zip](https://github.com/rlguy/Blender-FLIP-Fluids-Beta/files/1869678/Fluid.Test.07.zip)

**Comment by rlguy** [comment]
This was an error caused by a simple typo in the code. This fix will be included in the next version (**0.1.4b**). Unfortunately there is no workaround to this for this scene. I have added a custom build that includes this fix to your Dropbox folder.
